test/augmentation/test_augmentation.py

```diff
@@ -2551,3 +2552,159 @@ def test_smoke(self, device, dtype):
+
+                              [[0.2621, 0.4020, 0.2245, 0.0472],
+                               [0.3101, 1.0000, 1.0000, 1.0000],
+                               [0.7842, 0.3631, 0.8711, 0.0495],
+                               [0.2067, 0.3609, 1.0000, 1.0000]]],
+
+
+                             [[[0.3736, 0.5783, 0.0666, 0.0994],
+                               [0.2314, 0.4774, 0.3690, 0.6749],
+                                 [0.3430, 0.4760, 0.2627, 0.3024],
+                                 [0.0168, 0.1272, 0.2213, 0.3904]],
+
+                              [[0.6977, 0.8000, 0.1610, 0.2823],
+                                 [0.6816, 0.9152, 0.3971, 0.8742],
+                                 [0.4194, 0.5529, 0.9527, 0.0362],
+                                 [0.1852, 0.3734, 0.3051, 0.9320]],
+
+                              [[0.2621, 0.4020, 0.2245, 0.0472],
+                                 [0.3101, 1.0000, 1.0000, 1.0000],
+                                 [0.7842, 0.3631, 0.8711, 0.0495],
+                                 [0.2067, 0.3609, 1.0000, 1.0000]]]],
+                            device=device,
+                            dtype=dtype)
+
+    def _get_input(self, device, dtype):
+        return torch.tensor([[[[0.4963, 0.7682, 0.0885, 0.1320],
+                               [0.3074, 0.6341, 0.4901, 0.8964],
+                               [0.4556, 0.6323, 0.3489, 0.4017],
+                               [0.0223, 0.1689, 0.2939, 0.5185]],
+
+                              [[0.6977, 0.8000, 0.1610, 0.2823],
+                               [0.6816, 0.9152, 0.3971, 0.8742],
+                               [0.4194, 0.5529, 0.9527, 0.0362],
+                               [0.1852, 0.3734, 0.3051, 0.9320]],
+
+                              [[0.1759, 0.2698, 0.1507, 0.0317],
+                               [0.2081, 0.9298, 0.7231, 0.7423],
+                               [0.5263, 0.2437, 0.5846, 0.0332],
+                               [0.1387, 0.2422, 0.8155, 0.7932]]]],
+                            device=device, dtype=dtype)
+
+    def test_planckian_jitter_blackbody(self, device, dtype):
+        torch.manual_seed(0)
+        f = PlanckianJitter(select_from=1)
+        input = self._get_input(device, dtype)
+        expected = self._get_expected_output_blackbody(device, dtype)
+        assert_close(f(input), expected, atol=1e-4, rtol=1e-5)
+
+    def test_planckian_jitter_cied(self, device, dtype):
+        torch.manual_seed(0)
+        f = PlanckianJitter(mode='CIED', select_from=1)
+        input = self._get_input(device, dtype)
+        expected = self._get_expected_output_cied(device, dtype)
+        assert_close(f(input), expected, atol=1e-4, rtol=1e-5)
+
+    def test_planckian_jitter_batch(self, device, dtype):
+        torch.manual_seed(0)
+        input = self._get_input(device, dtype).repeat(2, 1, 1, 1)
+
+        select_from = torch.LongTensor([1, 2, 24])
```

Comment:
```suggestion
        select_from = [1, 2, 24]
```
Now that I see it all together, wondering if we should keep the high level interface free of the pytorch backend towards our plan to support other frameworks  /cc @shijianjian

Reply:
Do you mean, we should allow `Tensor` as input, So for `select_from` variable, I followed this API signature, which is also used in `color_jitter`.
https://github.com/kornia/kornia/blob/15fb1aebd8f9e16af61efb130efb004f7a7b7e20/kornia/augmentation/_2d/intensity/color_jitter.py#L61

Reply:
right, apart that the main reason to use Tensor is to differentiate, but in this case it doesn't make sense. Indices are not backpropagable.

Reply:
ah ok, made changes for the same!